src/praisonai/praisonai/tool_resolver.py

```
@@ -516,6 +516,60 @@ def _extract_tool_classes(self, module):
+        
+        try:
+            from praisonaiagents.toolsets import resolve_toolsets
+            
+            # Resolve toolset names to tool names
+            tool_names = resolve_toolsets(toolset_names)
+            logger.debug(f"Resolved toolsets {toolset_names} to tools: {tool_names}")
+            
+            # Resolve tool names to callables
+            return self.resolve_many(tool_names)
+            
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)